packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -735,7 +736,13 @@ export function createAPIFactory(
             },
             registerEditSessionIdentityProvider(scheme: string, provider: theia.EditSessionIdentityProvider) {
                 return workspaceExt.$registerEditSessionIdentityProvider(scheme, provider);
-            }
+            },
+            /**
+             * @stubbed
+             * This is a stub implementation, that should minimally satisfy vscode built-in extensions 
```

Comment:
There seems to be a trailing whitespace that is failing the `lint`:

```suggestion
             * This is a stub implementation, that should minimally satisfy vscode built-in extensions
```